pinot-tools/src/main/java/org/apache/pinot/tools/perf/PerfBenchmarkDriver.java

```diff
@@ -296,24 +295,6 @@ public void configureTable(String tableName, List<String> invertedIndexColumns,
     _helixResourceManager.addTable(tableConfig);
   }
 
-  private void uploadIndexSegments()
```

Comment:
Why do we remove this?

Reply:
`IndexDirectory` is not used in `PerfBenchmarkRunner` which is the wrapper around `PerfBenchmarkDriver`. We use `dataDir` + `tableName` instead. 

I added the code that's removing `IndexDirectory` configuration.